packages/table/src/Cell/Cell.tsx

```diff
@@ -2,21 +2,33 @@ import React from 'react';
 
 import { cx } from '@leafygreen-ui/emotion';
 
+import { useTableContext } from '../TableContext';
+
 import {
   alignmentStyles,
   baseCellStyles,
   basicCellStyles,
   cellContentContainerStyles,
+  disableAnimationStyles,
 } from './Cell.styles';
 import { CellProps } from '.';
 
-const Cell = ({ className, align, children, ...rest }: CellProps) => (
-  <td className={cx(baseCellStyles, basicCellStyles, className)} {...rest}>
-    <div className={cx(cellContentContainerStyles, alignmentStyles(align))}>
-      {children}
-    </div>
-  </td>
-);
+const Cell = ({ className, align, children, ...rest }: CellProps) => {
+  const { disableAnimations } = useTableContext();
```

Comment:
Do we have guidance on whether we prefer `disableXYZ` vs `enableXYZ` prop names? @bruugey

Reply:
I don't see anything in the [styleguide](https://github.com/mongodb/leafygreen-ui/blob/main/STYLEGUIDE.md#variable-naming). Should we?

Reply:
if the question is should we the answer is yes!!

Reply:
Does anyone know if this is an established standard anywhere? My understanding is that variables should never be set up for the user to have double-negatives (i.e. `isNotEditable` would be a bad variable name since the user might have to use `!isNotEditable` which reads as a double negative), but I'm not personally aware of anything against words like `disabled`.

I went with `disableAnimations` because animations are enabled by default we would generally recommend using animations. Setting `disableAnimations` to `true` feels more like we're disabling something that should be used instead of setting `enableAnimations` to `false`, which feels more like we're just electing to not use something that's optional.

If we like the thought process I had above, I can raise a separate PR to add it to our styleguide!

Reply:
You could argue that `!disabled` could also read as a double negative (though it's not quite as clear-cut) 
I'm fine with leaving it as-is, since we use `disabled` as a prop name elsewhere